cli/pkg/local/app.go

```diff
@@ -329,10 +334,10 @@ func (a *App) Serve(httpPort, grpcPort int, enableUI, openBrowser, readonly bool
 		return runtimeServer.ServeHTTP(ctx, func(mux *http.ServeMux) {
 			// Inject local-only endpoints on the server for the local UI and local backend endpoints
 			if enableUI {
-				mux.Handle("/", web.StaticHandler())
+				mux.Handle("/", runtimeserver.OtelHandler(web.StaticHandler()))
 			}
-			mux.Handle("/local/config", a.infoHandler(inf))
-			mux.Handle("/local/track", a.trackingHandler(inf))
+			mux.Handle("/local/config", runtimeserver.OtelHandler(a.infoHandler(inf)))
+			mux.Handle("/local/track", runtimeserver.OtelHandler(a.trackingHandler(inf)))
```

Comment:
These are static endpoints, we can remove otel tracking here. The runtime endpoints are the only ones that really matter